src/commonmark/en/content/user/importexport-app.md

```diff
@@ -33,24 +33,16 @@ which can be found below.
 
 <!--DHIS2-SECTION-ID:import_progress_logger-->
 
-No matter what you import ("Metadata", "Data", "Events" or "GML" data),
-you can always view the progress of the import by opening the logger at the
-bottom of the screen.
-
-* To open the logger, click on the "arrow up" icon on the right side:
-
-![](resources/images/import_export/logger-open.png)
-
-* To close the logger, click on the "arrow down" icon in the top-right corner:
-
-![](resources/images/import_export/logger-close.png)
+No matter what you import ("Data", "Events", "GML", "Metadata" or
+"Tracked Entity Instances" data), you can always view the progress of
+the import by looking at the "Job Summary" at the top of the page.
 
 ### Import Summaries
 
 <!--DHIS2-SECTION-ID:metadata_import_summaries-->
 
 On import request completion, we show import summaries above the import form.
-Any conflicts or errors are shown in a paginated table under totals.
+Any conflicts or errors are shown in a table.
```

Comment:
Is this change necessary? Can you be more specific?

Reply:
Removing `paginated` would've been enough, I think.
What do you think about?:
> On import request completion, we show import summaries above the
import form. Any conflicts or errors are shown in a table under the
overview summary for the import.

Reply:
lgtm. I would maybe use "in the table" instead of "a table", but it's better to confirm with our native eng speakers 👍